src/components/Waveform/WaveformTypes.ts

```diff
@@ -10,6 +10,8 @@ interface BaseWaveform {
   containerStyle?: StyleProp<ViewStyle>;
   waveColor?: string;
   mode: StaticOrLive;
+  onCurrentProgressChange?: (currentProgress: number, songDuration: number) => void;
```

Comment:
Do we need to add this to BaseWaveform or StaticWaveform?. Shouldn't it be just for static waveform. for live, we will not have a song duration.

Reply:
For live waveform as well we are getting song duration

Reply:
Done, Please check